libraries/objectDefaultFiles/object.js

```diff
@@ -2156,6 +2157,23 @@
             });
         }
 
+        /**
+         * Get the user's name and any other details about their session that the app knows
+         */
+        this.getUserDetails = function() {
+            postDataToParent({
+                getUserDetails: true
+            });
+            return new Promise((resolve, _reject) => {
+                spatialObject.messageCallBacks.userDetailsResult = function (msgContent) {
+                    if (typeof msgContent.userDetails !== 'undefined') {
+                        resolve(msgContent.userDetails);
+                        delete spatialObject.messageCallBacks['userDetailsResult']; // only trigger it once
```

Comment:
This can be `spatialObject.messageCallBacks.userDetailsResult` without the quoting